tests/aws/services/route53resolver/test_route53resolver.py

```diff
@@ -85,6 +86,57 @@ def setup_resources(self, aws_client, cleanups):
 
         yield vpc, subnet1, subnet2, security_group
 
+    @pytest.fixture(scope="function")
+    def route53resolver_create_firewall_rules(self, aws_client, cleanups):
+        """Set up the following resources:
+        - 1 firewall-rule-group
+        - 4 firewall-domain-lists
+        - 4 firewall-rules
+        """
+        firewall_rule_group_name = f"fw-rule-group-{short_uid()}"
+        rule_group_response = aws_client.route53resolver.create_firewall_rule_group(
+            Name=firewall_rule_group_name
+        )
+        cleanups.append(
+            lambda rule_group_id=rule_group_response["FirewallRuleGroup"][
+                "Id"
+            ]: aws_client.route53resolver.delete_firewall_rule_group(
+                FirewallRuleGroupId=rule_group_id
+            )
+        )
+        # Parameters for creating resources
+        priorities = [1, 2, 3, 4]
+        actions = [Action.ALLOW, Action.ALERT, Action.ALERT, Action.ALLOW]
+
+        for action, priority in zip(actions, priorities):
+            domain_list_response = aws_client.route53resolver.create_firewall_domain_list(
+                Name=f"fw-domain-list-{short_uid()}"
+            )
+            cleanups.append(
+                lambda domain_list_id=domain_list_response["FirewallDomainList"][
+                    "Id"
+                ]: aws_client.route53resolver.delete_firewall_domain_list(
+                    FirewallDomainListId=domain_list_id
+                )
+            )
+            _ = aws_client.route53resolver.create_firewall_rule(
```

Comment:
nit: Since we do not collect any response here, we can directly remove the left hand side and write `aws_client.route53resolver.create_firewall_rule(...)`